react-app/components/ListImages/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+
+import ImageItem from './components/ImageItem';
+import './styles.css';
+
+function ListImages(props) {
+  const { list } = props;
+
+  return (
+    <>
+      <div>
+        <h3 className="xblock-images-gallery__list-title">To upload</h3>
```

Comment:
^ Same comment as [in here](https://github.com/eduNEXT/xblock-imagesgallery/pull/4/files#r1316306512)